pkg/oauth/apis/oauth/types.go

```diff
@@ -36,6 +36,10 @@ type OAuthAccessToken struct {
 	// RefreshToken is the value by which this token can be renewed. Can be blank.
 	RefreshToken string
 
+	IdentityName string
```

Comment:
Add comment explaining what the field represents, what format/values are allowed, if it's required or optional. [example](https://github.com/kubernetes/kubernetes/blob/ea2f0be58f3d0c875ebdbbd4351d5a9143a55318/staging/src/k8s.io/client-go/tools/clientcmd/api/types.go#L95-L97)